### PR TITLE
Doc directives

### PR DESCRIPTION
**Pull request by bcoca**
##### SUMMARY

actually document directives page

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

docs
##### ANSIBLE VERSION

```
2.3
```

### CODE REVIEW

File: docs/docsite/directive_desc.yml
Comment (gundalow) [inline docs/docsite/directive_desc.yml:2]: `A string of flag(s) (white space separated)...

File: docs/docsite/directive_desc.yml
Comment (gundalow) [inline docs/docsite/directive_desc.yml:4]: User that you 'become' after using privilege escalation on the remote system, the ``remote_user`` user must have permissions to become this user.

File: docs/docsite/directive_desc.yml
Comment (gundalow) [inline docs/docsite/directive_desc.yml:10]: s/It does not affect connection errors./Connection errors will always cause an error and stop the playbook run for the  machine./


Not sure what the best wording for "stop the playbook run for the  machine", feel free to change

File: docs/docsite/directive_desc.yml
Comment (gundalow) [inline docs/docsite/directive_desc.yml:11]: Allows tasks to have a human readable name, which is best practice. The name can be used as an identified when uses with ``--start-at-task` or for `handlers:`.

File: docs/docsite/directive_desc.yml
Comment (gundalow) [inline docs/docsite/directive_desc.yml:12]: Also referred to as the "login user".

File: docs/docsite/directive_desc.yml
Comment (gundalow) [inline docs/docsite/directive_desc.yml:19]: ``check_mode``

**Review by gundalow — CHANGES_REQUESTED** [review]
Looks good, thanks for documenting this. Just a few suggestions of wording, though I'll let Scott have the final say.

**Comment by bcoca** [comment]
once merged, changing the docs is just editing the yaml file, so i'm going to leave as is for now,  you guys can modify to perfection afterwards.

**Comment by gundalow** [comment]
> once merged, changing the docs is just editing the yaml file, so i'm going to leave as is for now, you guys can modify to perfection afterwards.

Sounds good idea.

File: docs/docsite/directive_desc.yml
Comment (dharmabumstead) [reply docs/docsite/directive_desc.yml:10]: Suggest using 'Boolean that indicates..." rather than "Boolean that controls..."

File: docs/docsite/directive_desc.yml
Comment (dharmabumstead) [reply docs/docsite/directive_desc.yml:11]: +1 @gundalow

File: docs/docsite/directive_desc.yml
Comment (dharmabumstead) [reply docs/docsite/directive_desc.yml:19]: s/Boolean that controls/Boolean that indicates...

Example: 
Boolean that indicates whether or not a log is create.

**Review by dharmabumstead — CHANGES_REQUESTED** [review]

**Pull request merged**
